flake.nix

```diff
@@ -312,7 +317,9 @@
         };
       };
 
-    addShellCheck = name: package: exec: self: super: {
+    addShellCheck = name: package: addShellCheck' name (_: package);
```

Comment:
I think it should be:
```nix
addShellCheck = name: package: addShellCheck' name (_: _: package);
```
since the lambda will be called with two argouments on line [336](https://github.com/Liqwid-Labs/liqwid-nix/pull/12/files#diff-206b9ce276ab5971a2489d75eb1b12999d4bf3843b7988cbe8d687cfde61dea0R336).

Reply:
Yeah, that's the point. Basically `addShellCheck'` is a version that passes `system`, and `addShellCheck` is one that doesn't. What this is doing is just throwing away `system`.

Reply:
Oh, I see.